templates/taxi/driver_list.html

```diff
@@ -12,14 +16,22 @@
         <th>First name</th>
         <th>Last name</th>
         <th>License number</th>
+        <th>Delete</th>
+
       </tr>
     {% for driver in driver_list %}
       <tr>
         <td>{{ driver.id }}</td>
         <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
         <td>{{ driver.first_name }}</td>
         <td>{{ driver.last_name }}</td>
-        <td>{{ driver.license_number }}</td>
+        <td><a href="{% url 'taxi:driver-update' driver.id%}">{{ driver.license_number }}</a></td>
+        <td>
+          <a style="color: red"
+                href="{% url 'taxi:driver-delete' pk=driver.id %}">
```

Comment:
The URL in the href attribute should end with a slash to follow the checklist guideline: 'Don't forget to add / at the end of the URLs.'